plugins/kotlin-as-java/src/main/kotlin/signatures/JavaSignatureProvider.kt

```diff
@@ -196,6 +196,7 @@ class JavaSignatureProvider internal constructor(ctcc: CommentsToContentConverte
         is Star -> operator("?")
 
         is Nullable -> signatureForProjection(p.inner)
+        is DefinitelyNonNullable -> signatureForProjection(p.inner)
```

Comment:
Can you also add tests for kotlin-as-java? There are some signature-related ones in `KotlinAsJavaPluginTest` already